cardano-db-sync/src/Cardano/DbSync/Config/Cardano.hs

```diff
@@ -131,7 +133,7 @@ mkProtocolInfoCardano ge shelleyCred =
         (Consensus.ProtocolTransitionParamsShelleyBased () $ dncMaryHardFork dnc)
         (Consensus.ProtocolTransitionParamsShelleyBased alonzoGenesis $ dncAlonzoHardFork dnc)
         (Consensus.ProtocolTransitionParamsShelleyBased () $ dncBabbageHardFork dnc)
-        (Consensus.ProtocolTransitionParamsShelleyBased (ConwayGenesis (GenDelegs mempty)) Consensus.TriggerHardForkNever) -- TODO: Conway Fix
+        (Consensus.ProtocolTransitionParamsShelleyBased (ConwayGenesis (GenDelegs mempty)) (Consensus.TriggerHardForkAtEpoch (EpochNo 34))) -- TODO: Conway Fix
```

Comment:
maybe a little note as to why hard fork at epoch 34?

Reply:
Yes this needs to change. It's hardcoded 34 for sanchonet, but needs to be parsed from the config file before this is merged.

Reply:
Cool thanks, so it's usually an option in the config file